docs/storage-node-configuration.md

```diff
@@ -0,0 +1,406 @@
+|---------------------------|-------------------------------|---------------|---------------------------------------------------------------------------|
+| `endpoint`                | `[]string`                    | empty         | Address that service listener binds to.                                   |
+| `tls`                     | [TLS config](#tls-subsection) |               | Address that control service listener binds to.                           |
+
+## `tls` subsection
+
+| Parameter             | Type     | Default value | Description                                                               |
+|-----------------------|----------|---------------|---------------------------------------------------------------------------|
+| `enabled`             | `bool`   | `false`       | Address that control service listener binds to.                           |
+| `certificate`         | `string` |               | Path to the TLS certificate.                                              |
+| `key`                 | `string` |               | Path to the key.                                                          |
+| `use_insecure_crypto` | `bool`   | `false`       | If true, ciphers considered insecure by Go stdlib are allowed to be used. |
+
+# `pprof` section
+
+Contains configuration for the `pprof` profiler.
+
+| Parameter          | Type       | Default value | Description                             |
+|--------------------|------------|---------------|-----------------------------------------|
+| `enabled`          | `bool`     | `false`       | Flag to enable the service.             |
+| `address`          | `string`   |               | Address that service listener binds to. |
+| `shutdown_timeout` | `duration` | `30s`         | Time to wait for a graceful shutdown.   |
+
+
+# `prometheus` section
+
+Contains configuration for the `prometheus` metrics service.
+
+| Parameter          | Type       | Default value | Description                             |
+|--------------------|------------|---------------|-----------------------------------------|
+| `enabled`          | `bool`     | `false`       | Flag to enable the service.             |
+| `address`          | `string`   |               | Address that service listener binds to. |
+| `shutdown_timeout` | `duration` | `30s`         | Time to wait for a graceful shutdown.   |
+
+# `logger` section
+
+```yaml
+logger:
+  level: info
+```
+
+Contains logger parameters.
+
+| Parameter | Type     | Default value | Description                                                                                       |
+|-----------|----------|---------------|---------------------------------------------------------------------------------------------------|
+| `level`   | `string` | `info`        | Logging level.<br/>Possible values:  `debug`, `info`, `warn`, `error`, `dpanic`, `panic`, `fatal` |
+
+# `contracts` section
+
```

Comment:
We should mention explicitly that we don't need to set hashes here, everything should be read from the NNS, by default.

Reply:
Fixed